src/accelerate/utils/dataclasses.py

```diff
@@ -1098,6 +1098,7 @@ def __post_init__(self):
         if self.zero3_init_flag and not self.hf_ds_config.is_zero3():
             warnings.warn("DeepSpeed Zero3 Init flag is only applicable for ZeRO Stage 3. Setting it to False.")
             self.zero3_init_flag = False
+        self.enabled = False
```

Comment:
perhaps add a short note to tell where it gets enabled? just an idea if it resonates

Reply:
agreed :)